jaggr-core/src/main/java/com/ibm/jaggr/core/impl/modulebuilder/javascript/JavaScriptModuleBuilder.java

```diff
@@ -665,7 +669,9 @@ protected String moduleNameIdEncodingEndLayer(HttpServletRequest request, List<I
 			  .append(");"); //$NON-NLS-1$
 		}
 		// Finally, close the scoping function and invoke it.
-		sb.append("})();"); //$NON-NLS-1$
+		if (!noAMD) {

```

Comment:
Should we put a comment here saying why we don't close the scope here if it **is** noAMD?
like...  will we not have a scope to close/invoke if noAMD is true?